health/micro-ui/web/console/Dockerfile

```
@@ -0,0 +1,30 @@
+#FROM ghcr.io/egovernments/alpine-node-builder-14:yarn AS build
+RUN apk update && apk upgrade
+RUN apk add --no-cache git>2.30.0
+ARG WORK_DIR
+WORKDIR /app
+ENV NODE_OPTIONS "--max-old-space-size=4792"
+
+COPY ${WORK_DIR} .
+RUN ls -lah
+
+#RUN node web/envs.js
+RUN cd web/ \
```

Hadolint (2.12.0)

[warning] Use WORKDIR to switch to a directory

(DL3003)